scss/input/_layout.scss

```diff
@@ -191,6 +199,122 @@
         }
     }
 
+    // Forms
+    $form-spacer: $padding-x * 2 !default;
```

Comment:
$padding-x / $padding-y are used for margins in other places, too. I don't think we need additional variable here